env-example

```diff
@@ -4,3 +4,8 @@
 
 # prx logo
 LOGO_NAME=
+# What url to feed requests to 
+FEED_PROXY_URL=
+
+# What set of data sources should we use for fetching information?
+DATA_SOURCES=
```

Comment:
Yeah, the idea here is that we could come up with some way in the env to set up the set of adapters to be used, but it wound up hardcoded.